client/app/queue/mtv/MTVJudgeDisposition.jsx

```diff
@@ -36,18 +40,29 @@ const vacateTypeText = (val) => {
 };
 
 const formatInstructions = ({ disposition, vacateType, hyperlink, instructions }) => {
-  const parts = [`I am proceeding with a ${DISPOSITION_TEXT[disposition]}.`];
+  const parts = [`${MTV_TASK_INSTRUCTIONS} ${DISPOSITION_TEXT[disposition].replace(/(^\w{1})|(\s+\w{1})/g, (letter) => letter.toUpperCase())}\n`];
```

Comment:
https://github.com/department-of-veterans-affairs/caseflow/blob/76dd09c8b90826fbcad1eb720ffd0bc4d83e0e28/client/app/util/StringUtil.js#L50-L53

I think the `titleCase` function in `StringUtil` should be able to accomplish the same task.

```suggestion
  const parts = [`${MTV_TASK_INSTRUCTIONS} ${StringUtil.titleCase(DISPOSITION_TEXT[disposition])}\n`];
```